cdap-archetypes/cdap-app-archetype/src/main/resources/archetype-resources/src/main/java/HelloWorld.java

```diff
@@ -98,7 +98,7 @@ public void process(StreamEvent event) {
   }
 
   /**
-   * A {@link Service} that creates a greeting using a user's name.
+   * A Service that creates a greeting using a user's name.
```

Comment:
We do...but as we don't run Javadocs on the `examples`, having this in the examples just clutters up the comments. We haven't added links elsewhere in these examples to our Javadocs.